src/transformers/generation/beam_constraints.py

```diff
@@ -48,10 +48,13 @@ def test(self):
     @abstractmethod
     def advance(self):
         """
-        When called, returns the token that would take this constraint one step closer to being fulfilled.
+        When called, returns the token(s) that would take this constraint one step closer to being fulfilled.
 
         Return:
-            token_ids(`torch.tensor`): Must be a tensor of a list of indexable tokens, not some integer.
+            token_ids (Union[int, List[int], None]): 
```

Comment:
Trailing whitespace is causing the doc-builder to fail

```suggestion
            token_ids (Union[int, List[int], None]):
```